metadata_tools/geometry_support.py

```
@@ -969,6 +1246,63 @@ def __init__(self, input_dir, output_dir,
+            event_key = column_desc[0]
+            if len(column_desc) > 2:
+                format = ALT_FORMAT_DICT[(event_key[0], column_desc[2])]
+            else:
+                format = FORMAT_DICT[event_key[0]]
+
+            # Save label overrides for this column
+            (_,_,_,_,_, null_value, valid_minimum, valid_maximum) = format
+            override = {'NULL_VALUE':    null_value,
+                        'VALID_MINIMUM': valid_minimum,
+                        'VALID_MAXIMUM': valid_maximum, 
+                       }
```

Flake8 (7.3.0)

[error] closing bracket does not match visual indentation

(E124)